app/assets/javascripts/controllers/middleware_topology/middleware_topology_controller.js

```diff
@@ -99,7 +99,16 @@ function MiddlewareTopologyCtrl($scope, $http, $interval, $location, topologySer
                 class_name = "middleware_deployment";
                 break;
             case "MiddlewareServer":
-                class_name = "middleware_server";
+                switch (d.item.product) {
+                    case "Hawkular":
+                        class_name = "vendor-hawkular";
```

Comment:
can you please explain this use case? a server of type hawkular which is identical to the provider type itself? I'm just thinking of "here's a topology of a provider with hawkular icon, and another entity with the same icon is connected to it" - I'm not clear on how to explain it.